docs/writing-docs/doc-block-argstable.md

```diff
@@ -48,10 +48,11 @@ To use the `ArgsTable` in [DocsPage](./docs-page.md#component-parameter), export
 
 If you need, you can also include the `ArgsTable` block in your MDX stories. Below is a condensed table of available options and examples:
 
-| Option  | Description                                                                                         |
-| ------- | --------------------------------------------------------------------------------------------------- |
-| `of`    | Infers the args table from the component. <br/> `<ArgsTable of={MyComponent} />`                    |
-| `story` | Infers the args table based on a story. <br/> `<ArgsTable story="example-mycomponent--my-story" />` |
+| Option        | Description                                                                                         |
+| -------       | --------------------------------------------------------------------------------------------------- |
+| `of`          | Infers the args table from the component. <br/> `<ArgsTable of={MyComponent} />`                    |
+| `story`       | Infers the args table based on a story. <br/> `<ArgsTable story="example-mycomponent--my-story" />` |
+| `hideColumns` | An array of columns to hide. Hidable Columns: `descrption`, `default`, `control`. <br/> `<ArgsTable of={MyComponent} hideColumns={['default', 'description']}` |
```

Comment:
@sir-captainmorgan21, we have a bit of room for improvement here. If you wouldn't mind updating the table to the following:

```md
| Option        | Description                                                                                                                                                                                |
| ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
| `of`          | Infers the args table from the component. <br/> `<ArgsTable of={MyComponent} />`                                                                                                           |
| `story`       | Infers the args table based on a story. <br/> `<ArgsTable story="example-mycomponent--my-story" />`                                                                                        |
| `hideColumns` | Toggles visibility for the args table columns.<br/>Available options: `description`, `default`, `control`. <br/> `<ArgsTable of={MyComponent} hideColumns={['default', 'description']} />` |
```

Also, if you are ok, could you create a small code snippet file in the `docs/snippets/common` directory with a small example showcasing the work you did and link below in the `<CodeSnippets />` component? Turning it into:

```js
<CodeSnippets
  paths={[
    'common/component-story-mdx-argstable-block-for-component.with-component.mdx.mdx',
    'common/component-story-mdx-argstable-block-for-story.with-story.mdx.mdx',
    'common/component-story-mdx-argstable-hide-columns.with-story.mdx.mdx',  //👈 Your snippet file here
  ]}
/>
```

With it, we can provide a bit more clarity on the documentation.

If you have trouble adding the snippets, [this documentation](https://storybook.js.org/docs/react/contribute/new-snippets) provides you with the context you need, and if you have any questions regarding the documentation changes, feel free to ping me and I'll gladly answer them.

Reply:
Ill knock these out. Thanks!